tests/lib/rules/jsx-no-literals.js

```diff
@@ -351,37 +366,49 @@ ruleTester.run('jsx-no-literals', rule, {
       errors: [{message: stringsMessage('`Test`')}]
     }, {
       code: '<Foo bar={`Test`} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
       errors: [{message: stringsMessage('`Test`')}]
     }, {
       code: '<Foo bar={`${baz}`} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
       errors: [{message: stringsMessage('`${baz}`')}]
     }, {
       code: '<Foo bar={`Test ${baz}`} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
       errors: [{message: stringsMessage('`Test ${baz}`')}]
     }, {
       code: '<Foo bar={`foo` + \'bar\'} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
       errors: [
         {message: stringsMessage('`foo`')},
         {message: stringsMessage('\'bar\'')}
       ]
     }, {
       code: '<Foo bar={`foo` + `bar`} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
       errors: [
         {message: stringsMessage('`foo`')},
         {message: stringsMessage('`bar`')}
       ]
     }, {
       code: '<Foo bar={\'foo\' + `bar`} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
       errors: [
         {message: stringsMessage('\'foo\'')},
         {message: stringsMessage('`bar`')}
       ]
+    }, {
+      code: '<Foo bar={\'bar\'} />',
+      options: [{noStrings: true, validateProps: true}],
+      errors: [
+        {message: stringsMessage('\'bar\'')}
+      ]
+    }, {
+      code: '<Foo event={() => {}} />',
```

Comment:
this rule is about *string* literals. why would a function value ever trigger an error?

Reply:
I suppose I went a little too far on this. Will remove the logic.